src/client.ts

```diff
@@ -235,62 +219,53 @@ export class Web3BrowsersMode extends DelphinusWeb3 {
     }
     id = await this.getNetworkId();
     console.log("switched", id, chainHexId);
-  }
-}
-
-export class Web3ProviderMode extends DelphinusWeb3 {
-  readonly monitorAccount: string;
-  readonly delphinusProvider: DelphinusProvider;
-
-  constructor(config: MonitorMode) {
-    super(new Web3(config.provider.provider));
-
-    this.delphinusProvider = config.provider;
-    this.monitorAccount = config.monitorAccount;
-    super.setDefaultAccount(config.monitorAccount);
+    return;
   }
 
-  async connect() {}
-
-  async close() {
-    await this.delphinusProvider.close();
+  // Wrapper for personal_sign method
+  async sign(message: string): Promise<string> {
+    let signer = await this.provider.getSigner();
+    return await signer.signMessage(message);
   }
-
-  async switchNet(
-    chainHexId: string,
-    chainName: string,
-    rpcSource: string
-  ): Promise<void> {}
-}
-
-export interface MonitorMode {
-  provider: DelphinusProvider;
-  monitorAccount: string;
 }
 
-async function withDelphinusWeb3<t>(
-  web3: DelphinusWeb3,
-  cb: (web3: DelphinusWeb3) => Promise<t>
+export async function withBrowserProvider<T>(
+  cb: (web3: DelphinusBrowserProvider) => Promise<T>
```

Comment:
rename all web3 name, as we are not useing web3 any more.

Also rename getWeb3Contract() function in line 58 as now contract is ether Contract, not web3